contracts/scripts/native_solc_compile_all_automation

```diff
@@ -77,3 +77,11 @@ compileContract tests/VerifiableLoadStreamsLookupUpkeep.sol
 compileContract tests/VerifiableLoadLogTriggerUpkeep.sol
 compileContract tests/AutomationConsumerBenchmark.sol
 compileContract tests/StreamsLookupUpkeep.sol
+
+SOLC_VERSION="0.8.19"
+
+solc-select install $SOLC_VERSION
+solc-select use $SOLC_VERSION
+export SOLC_VERSION=$SOLC_VERSION
+
+compileContract automation/upkeeps/LinkAvailableBalanceMonitor.sol
```

Comment:
why is this necessary? We usually only make go wrappers for contracts that the core node needs to interact with.

Reply:
it's there because I used this script to compile the contract and added it by mistake. Removing!